data/changelog/merlin/2014-08-23-merlin-2.0.0.md

```diff
@@ -14,7 +14,7 @@ changelog: |
   * automatic reloading of .merlin files (when they are update or created), it
     is no longer necessary to restart Merlin
   * introduced a small refactoring command: rename, who renames all occurences
-    of an identifier. See [here](http://yawdp.com/~host/merlin_rename.webm).
+    of an identifier. See [here](http://yawdp.com/~host/merlin_rename.webm - [1 Client error: Couldn't resolve host name]).
```

Comment:
```suggestion
    of an identifier. See [here](https://github.com/ocaml/merlin/blob/master/vim/merlin/doc/merlin.txt).
```
This is the only thing close I could find.

Reply:
I don't think that's what they were linking to, so I'm removing the link altogether.

```suggestion
    of an identifier.
```